src/loaders/jobs/taskfarmer/taskfarmer_common.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+import os
+import subprocess
+
+# directory containing the TaskFarmer job scripts under the root directory
+TASKFARMER_JOB_DIR = 'task_farmer_jobs'
+# file containing the information of each task
+TASK_INFO_FILE = 'task_info.jsonl'
```

Comment:
Seems like these two constants no longer need to be in common, they can just live in the manager

Reply:
👍

Reply:
:+1: